python-backend/app/api/mines/tailings/resources/tailings.py

```diff
@@ -47,9 +49,11 @@ def post(self, mine_tailings_storage_facility_guid=None):
 
             if is_mine_first_tsf:  
                 try: 
-                    tsf_required_documents = requests.get(current_app.config['DOCUMENT_MS_URL'] + url_for('documents_required_document_resource') + '?category=MINE_TAILINGS', 
+                    get_resp = requests.get(current_app.config['DOCUMENT_MS_URL'] + current_app.config['BASE_PATH'] + '/documents/required' + '?category=MINE_TAILINGS', 

```

Comment:
If we're going to end up doing this all over the place, maybe move this to a constant or env file somewhere?

Reply:
This is the only place doing this kind of url construction, and it's still very WIP, when we find a pattern we can reliably repeat, definitely agree to pull it out to some kind of service/constants.

Reply:
For the time being, can you move it to the Top of the class/file just for cleanliness?